packages/fetchai/connections/p2p_stub/connection.py

```diff
@@ -78,20 +73,30 @@ async def send(self, envelope: Envelope):
 
         :return: None
         """
-
+        assert self.loop is not None, "Loop not initialized."
         target_file = Path(os.path.join(self.namespace, "{}.in".format(envelope.to)))
-        if not target_file.is_file():
-            target_file.touch()
-            logger.warn("file {} doesn't exist, creating it ...".format(target_file))
```

Comment:
Why removed?

Reply:
file created  on `open(, "ab")` if it can not be created at this point touch will fail as well, check `is_file` quite useless, cause if it's non file object it will fail on open